lambdas/build-stac/tests/conftest.py

```diff
@@ -42,3 +43,70 @@ def s3_created_bucket(s3_resource, mock_environment) -> Bucket:
 
     s3_bucket.create()
     yield s3_bucket
+
+
+@pytest.fixture
+def sample_assets():
+    return {
+        "cov_1-1.hdr": "s3://nasa-maap-data-store/file-staging/nasa-map/AfriSAR_UAVSAR_Ungeocoded_Covariance___1/uavsar_AfriSAR_v1-cov_coreg_fine_eighty_14047_16008_140_006_160225_cov_1-1.hdr",
+        "cov_1-1.bin": "s3://nasa-maap-data-store/file-staging/nasa-map/AfriSAR_UAVSAR_Ungeocoded_Covariance___1/uavsar_AfriSAR_v1-cov_coreg_fine_eighty_14047_16008_140_006_160225_cov_1-1.bin",
+    }
+
+
+@pytest.fixture
+def cmr_multi_asset_sample_event(sample_assets):
+    return events.CmrEvent(
+        **{
+            "collection": "AfriSAR_UAVSAR_Ungeocoded_Covariance",
+            "remote_fileurl": "s3://nasa-maap-data-store/file-staging/nasa-map/AfriSAR_UAVSAR_Ungeocoded_Covariance___1/uavsar_AfriSAR_v1-cov_coreg_fine_hsixty_14050_16015_140_009_160308_cov_1-1.hdr",
```

Comment:
Not sure where you're seeing external calls. Can you show me?

Reply:
I just understood that : 
- since it's a CMR event, we wouldn't be reading the file in itself anyways
- in addition, you're using the `patch` method to skip the `GranuleQuery` call.